src/Data/Text/Internal/Fusion.hs

```diff
@@ -103,7 +113,13 @@ reverseStream (Text arr off len) = Stream next (off+len-1) (betweenSize (len `sh
             n2 = A.unsafeIndex arr (i - 1)
 {-# INLINE [0] reverseStream #-}
 
--- | /O(n)/ Convert a 'Stream Char' into a 'Text'.
+-- | /O(n)/ Convert 'Stream` `Char' into a 'Text'.
```

Comment:
```suggestion
-- | /O(n)/ Convert `Stream` `Char` into a 'Text'.
```

Reply:
Let me change all links to use single quotes, I don't know what I was thinking.

Reply:
Changed all occurrences now.